pkg/autoscaler/autoscaler.go

```diff
@@ -360,8 +347,6 @@ var _ = Describe("[Feature:Machines] Autoscaler should", func() {
 		Expect(client.Create(ctx, clusterAutoscaler)).Should(Succeed())
 		cleanupObjects[clusterAutoscaler.GetName()] = clusterAutoscaler
 
-		By(fmt.Sprintf("Deriving Memory capacity from machine %q", existingMachineSets[0].Name))
-
```

Comment:
i think there used to be something here, but we are no longer using the `existingMachineSets` to derive the memory capacity.